subxt/src/config/mod.rs

```diff
@@ -12,6 +12,7 @@ mod default_extrinsic_params;
 mod extrinsic_params;
 
 pub mod polkadot;
+pub mod refine_params;
```

Comment:
I wouldn't export this mod I reckon; there's no real need since the only item it exports is below anyway :)